shared/reducers/config.js

```diff
@@ -112,6 +113,16 @@ export default function(state: Types.State = initialState, action: ConfigGen.Act
       const {globalError} = action.payload
       if (globalError) {
         logger.error('Error (global):', globalError)
+        if (
```

Comment:
i think we should move this into util/error and have some kind of transform there. it can return null if its this case and we'll ignore it

Reply:
if we move it to `convertToError`, the calling saga won't be able to tell that the RPC failed. is that what we want?

Reply:
we don't need to use convertToError per se, i'm just saying we should move this kind of logic to util/error

Reply:
@chrisnojima got it, will do